build_and_release.rb

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/ruby
+
+# This script builds the code, overriding the version number of the pom with
+# one of the form MAJOR.MINOR.BUILD. The major and minor version numbers are
+# taken from the pom file (see the BLUEFLOOD_VERSION maven parameter). The
+# build number should be passed in as the first and only command line argument
+# to the script. The script will then turn a snapshot version string, such as
+# "2.0-SNAPSHOT", into something suitable for a release build, like "2.0.36".
+#
+# Running maven from the command line (e.g. `mvn package`), instead of the
+# script, will still produce artifacts with the SNAPSHOT version string.
+
+require 'rexml/document'
+
+
+def get_version_number(build_number)
+  pom = REXML::Document.new(File.new("pom.xml"))
+  version = pom.elements["project/properties/BLUEFLOOD_VERSION"].text
+
+  if version.end_with? '${VERSION_SUFFIX}'
```

Comment:
@ChandraAddala 
Now that I look at it, I'm not sure what you mean. The default value of `BLUEFLOOD_VERSION` is `2.0.${VERSION_SUFFIX}`. The `VERSION_SUFFIX` part is provided by the build script (which leads to a release build), or is left as the default of `0-SNAPSHOT` (which leads to a snapshot). There are cases when both with and without the `-SNAPSHOT` suffix is acceptable. Are you proposing throwing an exception when `VERSION_SUFFIX` has the snapshot suffix in it? Because that property will already be overwritten just by running the build script.